.taskcluster.yml

```diff
@@ -7,6 +7,8 @@ metadata:
 tasks:
   - provisionerId: "{{ taskcluster.docker.provisionerId }}"
     workerType: "{{ taskcluster.docker.workerType }}"
+    scopes:
+        - "auth:aws-s3:read-only:taskcluster-raw-docs/*"
```

Comment:
afaict from my reading of the aws-sdk, even if it is public, the downloader still needs some sort of access key or the library yells about it.

I guess we could make it a static website as well and do the download over http? Or perhaps hopefully I'm just wrong about how to use the sdk without credentials.